packages/actionmenu/.storybook-storyshots/main.js

```diff
@@ -4,5 +4,12 @@ module.exports = {
   typescript: {
     check: true,
     reactDocgen: 'react-docgen-typescript'
+  },
+  webpackFinal: async (config, { configType }) => {
+    config.module.rules.push({
+      test: /\.mjs$/,
+      type: 'javascript/auto'
+    })
+    return config
   }
```

Comment:
we could explore adding this to the storybook-preset to DRY things up. that could come later